examples/python_model_optimizer/README.md

```diff
@@ -0,0 +1,55 @@
+# MIGraphX Model Optimizer
+This command line tool is meant to provide a simple way to optimize and serialize deep learning models for faster deployment. 
+
+## Basic Usage
+MIGraphX must first be built in order to use the model optimizer. Please refer to the top-level [README](/README.md) for instructions on how to build and install.
+
+
+```
+$ python3 optimize_model.py -h
+```
+
+```
+usage: optimize_model.py [-h] [--nchw] [-t {gpu,cpu,ref}] [-d] [-o] [-f] [-i] [-j] [-p OUTPUT_PATH] model
+
+Compile and serialize model offline for future deployment.
+
+positional arguments:
+  model                 Path to ONNX or TF Protobuf file
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --nchw                Treat Tensorflow format as nchw. (Default is nhwc)
+  -t {gpu,cpu,ref}, --target {gpu,cpu,ref}
+                        Compilation target
+  -d, --disable_fast_math
+                        Disable optimized math functions
+  -o, --offload_copy_off
```

Comment:
Probably, better to do `--disable-offload-copy` as its consistent with other flags.